packages/ui/primitives/document-dropzone.tsx

```diff
@@ -175,10 +187,10 @@ export const DocumentDropzone = ({
           </p>
 
           <p className="text-muted-foreground/80 mt-1 text-sm">
-            {disabled ? disabledMessage : 'Drag & drop your document here.'}
+            {disabled ? disabledMessage : 'Drag & drop your PDF here.'}
```

Comment:
Do we need to specify a pdf? Document is fine imo

Reply:
It's done to clarify that we only take in PDFs